corehq/apps/hqwebapp/static/registration/less/registration-main.less

```diff
@@ -187,3 +187,246 @@ p.validation-message-block {
 .sign-up-bubble {
   margin: 10px 0;
 }
+
+.no-touch .reg-start-container {
+  margin-top: 6vh;
+
+  @media(max-height: 770px) {
+    margin-top: 0;
+  }
+}
+
+.touch .reg-start-container {
+  margin-top: 8vh;
+
+  @media(max-height: 950px) {
+    margin-top: 5vh;
+  }
+
+  @media(max-height: 870px) {
+    margin-top: 0;
+  }
+}
+
+.reg-start-container > .well {
+  margin-top: 20px;
+  padding: 20px;
+  .border-top-radius(5px);
+  .border-bottom-radius(5px);
+
+  h2 {
+    text-align: center;
+    color: white;
+    margin-bottom: 20px;
+  }
+}
+
+.reg-start-container .check-list {
+  @_text-indent: 8px;
```

Comment:
Stylistic, but the `@_` is kind of funny-looking. Why not just `@text-indent`? Most of our LESS variables are "private", which I think is fairly obvious when you see them defined inside of a block.